xml/installation-installation-gui_installer.xml

```diff
@@ -8,51 +8,72 @@
+  process simpler, more accurate and less time consuming than manual
+  installation.
  </para>
  <para>
-  Using the installer, the series Cobbler and Ansible scripts that initiate the
-  installation of &kw-hos; are run for you behind the scenes.
+  Using information from the installer, Cobbler and Ansible scripts run behind
+  the scenes to initiate the installation of &kw-hos;.
  </para>
  <para>
-  The UI app populates the configuration files that the configuration processor
-  uses as input to its processes. Therefore you can inspect the files whenever
-  you like, or before reconfiguring your cloud.
+  The GUI app populates the configuration files that the configuration processor
+  uses as input to its processes. You can inspect the files at any time,
+  including just prior to actually configuring your cloud.
  </para>
  <para>
-  Note, the GUI installer will periodically save the configuration files during
-  the process and when you click the <guimenu>Deploy</guimenu> button at the
-  end of the process.
-  This also means that if you were to try to change anything in the files
-  themselves mid process, the values would be overwritten by values you have
-  entered in the GUI.
+  The GUI installer will periodically save the configuration files during
+  the process. When you click the <guimenu>Deploy</guimenu> button at the
+  end of the process, the complete configuration will be implemented. This
+  means that if you change anything directly in the files themselves during the
+  GUI installation process, those values will be overwritten by values you have
+  entered with the GUI. It also means that you can stop the GUI installation
+  process before clicking <guimenu>Deploy</guimenu> to see how the YAML files
+  have been configured.
  </para>
  <para>
-  These YAML configuration files are stored in a local git repository as well,
+  These YAML configuration files are stored in a local git repository,
   as explained in <xref linkend="using_git"/>.
  </para>
  <para>
-  When installing via the GUI, you begin with a model (or a cloud configuration
-  template) and edit the contents to reflect your environment.
+  When installing via the GUI, you begin with one of 8 input model examples
+  (essentially cloud configuration templates) and are guided step-by-step to
+  edit the contents to reflect your environment.
  </para>
  <para>
-  However, the full range of customizations is not possible via the GUI as some
-  configuration items are not exposed. Below is the list of what can and cannot
-  be changed:
+  The full range of customizations is not possible using the GUI. Some customizations require editing the yml files directly, which can be done in the "review configuration files" section of the GUI. Below is the list of what can be changed via GUI widgets, followed by a list of customizations that require direct editing of the configuration files to change.
```

Comment:
not sure if you need line breaks here or if the long line is ok... guess that depends on the doctool